src/translate-adapter-handlers.ts

```diff
@@ -194,16 +204,95 @@ async function translateIoPackage(): Promise<void> {
 	if (content.common.messages) {
 		console.log("Translate Messages");
 		for (const message of content.common.messages) {
-			console.log(`   Message: ${message.title.en}`);
+			console.log(gray(`   Message: ${message.title.en}`));
 			await translateNotExisting(message.title);
 			await translateNotExisting(message.text);
-			await translateNotExisting(message.linkText);
+			// test first if there is a linkText - since it's not mandatory
+			if (message.linkText) await translateNotExisting(message.linkText);
 		}
 	}
 	await writeJson(ioPackage, content, { spaces: 4, EOL });
 	console.log(`Successfully updated ${path.relative(".", ioPackage)}`);
 }
 
+async function loopJSON(content: object): Promise<object> {
+	const baseContent: object[] = [];
+	// read all existing language files
+	for (let n = 0; n < i18nBases.length; n++) {
+		if (existsSync(i18nBases[n])) {
+			baseContent.push(await readJson(i18nBases[n]));
+		}
+	}
+	// if there is no existing lang file create the default one
+	if (baseContent.length === 0)
+		baseContent[0] = await readJson("./admin/i18n/en/translations.json");
+	let baseContentChanged = false;
+	// iterate over the jsonConfig to find translatable entries
+	for (const [key, value] of Object.entries(content)) {
+		if (key === "i18n" && value === false) {
+			console.log(
+				"Info: i18n-switch is set to false; No translation of jsonConfig needed; Exiting.",
+			);
+			return content;
+		}
+		if (typeof value === "object") {
+			if (value.en) {
+				console.log(gray(`Translating: "${value.en}"`));
+				await translateNotExisting(value);
+				console.log(
+					yellow(
+						`Translated text object - but please consider changing it to: >"${key}": "${value.en}"< to be compliant with WebLate translation.`,
+					),
+				);
+			} else {
+				if (value.title || value.tooltip || value.label || value.text) {
+					const logText =
+						value.title ||
+						value.tooltip ||
+						value.label ||
+						value.text;
+					if (typeof logText === "string") {
+						for (let n = 0; n < baseContent.length; n++) {
+							// @ts-ignore
+							if (!baseContent[n][logText]) {
+								baseContentChanged = true;
+								// @ts-ignore
+								baseContent[n][logText] = logText;
```

Comment:
What this code does? it adds the value into all languages? becazse baseCOntent is the content of all trx files basically, or?